resources/test/valid/1_0_0/chainspec.toml

```diff
@@ -0,0 +1,137 @@
+[protocol]
+version = '1.0.0'
+
+[protocol.activation_point]
+era_id = 1
+
+[network]
+name = 'test-chain'
+timestamp = '2020-09-18T18:45:00Z'
```

Comment:
What's that? It has the same value as `network.timestamp` in the `0.9.0` chainspec.

Reply:
This file is just for testing parsing of the chainspec.  The actual values could be anything for these purposes.

Reply:
I was wondering what does the `timestamp` stands for. Previously, it was `genesis.timestamp`, which is more descriptive. Are we going to use `network.timestamp` to delay the start of the first era after the upgrade?

Reply:
I hadn't actually considered that, but I suppose we could use it that way.

As for the name, I'm open to changing it if others prefer too?  Maybe `network.start_time` would be more descriptive?